afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AppStorageListener.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.afs.storage;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface AppStorageListener {
+
+    void nodeCreated(NodeId id);
+
+    void nodeRemoved(NodeId id);
+
+    void attributeUpdated(NodeId id, String attributeName);
```

Comment:
Do we need to have the old and the value here?

Reply:
Maybe later on in a future version